metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/objects/ObjectToImport.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.importer.objects;
+
+import uk.ac.ebi.ampt2d.metadata.importer.objectImporters.ObjectsImporter;
+
+public interface ObjectToImport<OBJECT_TYPE> {
+
+    OBJECT_TYPE doImport(ObjectsImporter objectsImporter);
```

Comment:
I find this method name very confusing, as it seems that the object to import is importing `objectsImporter`. I suggest just calling it `accept` as it is the de facto standard in the visitor pattern.